nautilus_trader/common/component.pxd

```diff
@@ -284,6 +284,7 @@ cdef class MessageBus:
     cpdef bint is_subscribed(self, str topic, handler)
     cpdef bint is_pending_request(self, UUID4 request_id)
     cpdef bint is_streaming_type(self, type cls)
+    cpdef int num_subscribers(self, str pattern=*)
```

Comment:
This probably doesn't justify a separate method, because I think its enough to just be able to call
```
len(self.subscriptions(pattern))
```